docs/snakefiles/modularization.rst

```diff
@@ -199,6 +199,28 @@ Note that the second use statement has to use the original rule name, not the on
 In order to overwrite the rule ``some_task`` that has been imported with the first ``use rule`` statement, it is crucial to ensure that the rule is used with the same name in the second statement, by adding an equivalent ``as`` clause (here ``other_some_task``).
 Otherwise, you will have two versions of the same rule, which might be unintended (a common symptom of such unintended repeated uses would be ambiguous rule exceptions thrown by Snakemake).
 
+In addition, it is possible to load multiple modules dynamically by providing the ``name`` keyword.
+For example, by reading the module name from a config file or by iterating over several modules in a loop.
+For this, the module name is not specified directly after the ``module`` keyword, but by specifying the ``name`` parameter. 
+
+.. code-block:: python
+
+    for module, use_as in zip(['module1', 'module2'], ['module1_', 'module2_'])
+        module:
+            name: module
+            snakefile: f"{module}/Snakefile"
+            config: config[module]
+
+        use rule * from module as use_as*
```

Comment:
I think we should explicitly say here that use rule now has the following logic now:
1. it first tries to interpret the given identifier behind from as a module name. In that case, that name is interpreted as the prefix if used behind `as`.
2. If there is no module with that identifier as name, it will try to interpret the identifier as a variable name. In that case, it interprets any identifier used behind the as keyword as variable name as well, which includes eventually trailing underscores. Therefore one has to use a separate variable for those prefixes.